src/lib.rs

```diff
@@ -141,6 +144,28 @@ impl FixedBitSet
         }
     }
 
+    pub fn count_ones<T: IndexRange>(&self, range: T) -> usize
+    {
+        let start = range.start().unwrap_or(0);
+        let end = range.end().unwrap_or(self.length - 1);
+        // range makes sure that range.start <= range.end
+        assert!(end < self.length);
```

Comment:
Ah i see it now. I'm not sure why i wrote it incorrectly, but it's fixed now. Thanks for heads up!